cli/command/service/opts.go

```diff
@@ -510,7 +510,7 @@ func addServiceFlags(cmd *cobra.Command, opts *serviceOptions) {
 	flags.Var(&opts.restartPolicy.window, flagRestartWindow, "Window used to evaluate the restart policy")
 
 	flags.Uint64Var(&opts.update.parallelism, flagUpdateParallelism, 1, "Maximum number of tasks updated simultaneously (0 to update all at once)")
-	flags.DurationVar(&opts.update.delay, flagUpdateDelay, time.Duration(0), "Delay between updates")
+	flags.DurationVar(&opts.update.delay, flagUpdateDelay, time.Duration(0), "Delay between updates (Default none and Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h')")
```

Comment:
Agreed; I suggest to remove the `(or 'µs')` as well

    Delay between updates. Valid time units are 'ns', 'us', 'ms', 's', 'm', and 'h' (default none)


However, we try to make the output fit within 80~90-chars so that it fits in smaller consoles without wrapping, and that may still be too wide (138 characters);

```
      --update-delay duration              Delay between updates. Valid time units are 'ns', 'us', 'ms', 's', 'm', and 'h' (default none)
```

@dnephin I know this has come up before, but I think we should have a look if we can update Cobra to address these cases